org.cloudcompare.CloudCompare.yaml

```diff
@@ -231,6 +240,9 @@ modules:
       # see https://github.com/CloudCompare/CloudCompare/issues/848
       - type: patch
         path: patches/cc_constexpr_double.patch
+      # To be removed when bumping to 2.12
+      - type: patch
+        path: patches/qPDALIO.patch
```

Comment:
Is this patch seems to be from the upstream https://github.com/CloudCompare/CloudCompare/pull/1478. Do you think it's possible to just use a state from the upstream repo that contains this fix and is relatively stable? I would prefer this over carrying patches around that are already merged upstream.

Reply:
hmmm, we could use master but then it would be CloudCompare 2.12-beta

otherwise maybe we could get the commit from CloudCompare/CloudCompare#1478 cherry picked into the 2.11.x branch of CloudCompare and use that

Reply:
I don't mind switching to the beta, unless this adds more problems with dependencies etc. Otherwise, we can just carry the patches for now. Doing a new 2.11.4 release just for the flatpak is an overkill.

Reply:
Ok i'll try with the master branch

Reply:
Nice, we don't need any patches for CloudCompare. Thanks.